libtsuba/src/RDG.cpp

```diff
@@ -487,12 +487,10 @@ tsuba::RDG::DoMake(const katana::Uri& metadata_dir) {
 }
 
 katana::Result<tsuba::RDG>
-tsuba::RDG::Make(
-    const RDGMeta& meta, std::optional<uint32_t> host_to_load,
-    const std::vector<std::string>* node_props,
-    const std::vector<std::string>* edge_props) {
-  uint32_t host_being_loaded =
-      host_to_load.has_value() ? host_to_load.value() : Comm()->ID;
+tsuba::RDG::Make(const RDGMeta& meta, const struct RDGLoadOptions& opts) {
+  uint32_t host_being_loaded = opts.partition_to_load.has_value()
```

Comment:
You might enjoy: https://en.cppreference.com/w/cpp/utility/optional/value_or

Reply:
I did enjoy

Reply:
`HostID partition_to_load = ...` (until we add `PartitionID`)